### ISSUE TITLE
Add the 'useShortTags' option

### ISSUE THREAD

**Issue by agoat** [jakearchibald/svgomg#93]
In SVGO version >0.5.4 there is a paramter called `useShortTags` which is `true` by default. 
To make a svg compatible to old browsers it would be nice to have a option for this in the gui.

**Comment by jakearchibald** [comment]
Yeah, I need to find some time to bring SVGOMG up to date with SVGO. I'll try 😄
